src/crispy/flags.h

```diff
@@ -52,6 +52,17 @@ class flags
         _value &= ~static_cast<value_type>(flags._value);
     }
 
+    template<typename ...T>
+    [[nodiscard]] constexpr bool containsAll(T ...args)
+    {
+        return ((_value & args) && ...);
+    }
+
+    [[nodiscard]] constexpr bool contains(flag_type flag) const noexcept
+    {
+      return _value & flags<flag_type>{flag}.value();
+    }
+
```

Comment:
Instead of adding those two functions, we should fix the one below, that is:

```cpp
    /// Tests for existence of all given flags to be present.
    /// @return true if all flags are set in this flags set, false otherwise.
    [[nodiscard]] constexpr bool contains(flags<flag_type> flags) const noexcept
    {
        return (_value & flags.value()) == _flags.value();
    }
```

Reply:
Updated to this implementation